extensions/amp-ad-network-doubleclick-impl/amp-ad-network-doubleclick-impl-internal.md

```diff
@@ -35,7 +37,52 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Examples</strong></td>
-    <td>TBD</td>
+    <td><a href="#examples">See Example Section</a></td>
+  </tr>
+</table>
+
+### Supported Features
+
+<table>
+  <tr>
+    <td><strong>Feature</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Status</strong></td>
+  </tr>
+  <tr>
+    <td><a href="multi-size.md">Multi-size</a></td>
+    <td>Allows slots to specify multiple sizes.</td>
+    <td>Launched</td>
+  </tr>
+  <tr>
+    <td><a href="fluid.md">Fluid</a></td>
+    <td>Fluid slots do not require a pre-specified size, but will instead fill up the width of their parent container and adjust their height accordingly.</td>
+    <td>Beta</td>
+  </tr>
+  <tr>
+    <td><a href="doubleclick-rtc.md">Real Time Config</a></td>
+    <td>Allows Publishers to augment ad requests with targeting information that is retrieved at runtime.</td>
+    <td>Launched</td>
+  </tr>
+  <tr>
+    <td><a href="render-on-idle.md">Render on Idle</a></td>
+    <td>Allows slots 3-12 viewports down to render while the AMP scheduler is idle.</td>
+    <td>Launched</td>
+  </tr>
+  <tr>
+    <td><a href="refresh.md">Refresh</a></td>
+    <td>Enabled slots will periodically refetch new creatives.</td>
+    <td>Launched</td>
+  </tr>
+  <tr>
+    <td><a href="safeframe.md">Safeframe API</a></td>
+    <td>An API-enabled Iframe for facilitating communication between publisher page and creative content.</td>
+    <td>Launched</td>
+  </tr>
+  <tr>
+    <td><a href="sra.md">SRA: Single Request Architecture</a></td>
+    <td>When enabled, all eligible slots on the page will be serviced by a single ad request.</td>
+    <td>Beta</td>
   </tr>
 </table>
 
```

Comment:
Let's do an audit of the parameters and make sure we're clearer about what each means with examples (if not covered in features section).  For instance, targeting with a good example.  Also, why do we say anchors/Adhesion Units are not supported (we should link to amp-sticky-ad) and expandables (can use safeframe)?

Reply:
Added some descriptions to the attributes, and modified supported/unsupported DFP formats. Please proof read and give me notes.